README.md

```diff
@@ -39,7 +39,71 @@ proxy is required that runs in front of Kibana.
 The deployer enables the user to generate all of the necessary
 key/certs/secrets and deploy all of the components in concert.
 
-## Defining local builds
+#### Curator
+
+Curator allows the admin to remove old indices from Elasticsearch on a per-project
+basis.  The pod will read its configuration from a mounted yaml file that
+is structured like this:
+
+    $PROJECT_NAME:
+      $ACTION:
+        $UNIT: $VALUE
+
+    $PROJECT_NAME:
+      $ACTION:
+        $UNIT: $VALUE
+     ...
+
+* $PROJECT_NAME - the actual name of a project - "myapp-devel"
+* $ACTION - the action to take - currently only "delete"
+* $UNIT - one of "days", "weeks", or "months"
+* $VALUE - an integer for the number of units
+
+For example, using::
+
+    myapp-dev:
+     delete:
+       days: 1
+
+    myapp-qe:
+      delete:
+        weeks: 1
+     ...
+
+Every day, curator will run, and will delete indices in the myapp-dev
+project older than 1 day, and indices in the myapp-qe project older than 1
+week.
```

Comment:
Can we call out how to configure curating ops logs? I assume you just use project name `.operations` in the yaml but I think we need to say that.